core/src/rustic_ai/core/agents/utils/user_proxy_agent.py

```diff
@@ -89,6 +86,7 @@ def __init__(self):
         self.guild_requests_topic = UserProxyAgent.get_user_system_requests_topic(self.user_id)
 
         self.guilds_agents_ats: Dict[str, AgentTag] = {}
+        self._tag_pattern = self._build_tag_pattern()
```

Comment:
Why are we changing this? Any particular reason or benefit with the alternate apporach?

Reply:
This is to support tags with spaces. Earlier we were limiting tags to `\w` so for the message `@Echo Agent hello there. how are you ?` , where agent name is `Echo Agent`, the agent was not being identified as recipient